### DISCUSSION TITLE
could not find function "GetGRangesFromEnsDb"

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#802] [General]
I'm working Signac version (0.2.5). 
Certain commands are not being recognized 
could not find function "GetGRangesFromEnsDb"

Same issue with CreateChromatinAssay

Could you help with addressing this problem.

Thanks!

**Comment by timoast** [comment]
These functions are part of later Signac releases (>=1.0.0)

I suggest updating Signac to the latest release
